app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,13 @@
 
     <%= stylesheet_link_tag "application" %>
     <%= javascript_include_tag "application" %>
+    
+    <script type="text/javascript">
+      window.I18n = {
+        required_field: '<%= j t("form_validation.required_field") %>',
+        invalid_email: '<%= j t("form_validation.invalid_email") %>'
+      };
+    </script>
```

Comment:
This doesn't seem like a good idea. Are you going to preload any future translations into the main layout? Implement this only on the specific view that its needed